### PR TITLE
Update docu and reading of release area scenario

### PR DESCRIPTION
**Pull request by awirb**
* update docu (add in1DataPlots, statsPlots, update release scenario info)
* add options to call release area scenario with/without extension .shp 
* update com1DFACfg.ini

### CODE REVIEW

**Review by fso42 — CHANGES_REQUESTED** [review]

File: docs/moduleCom1DFA.rst [hunk 1]
```diff
@@ -31,9 +31,11 @@ and the following files are optional:
 
 The simulation settings area defined in the configuration file ``com1DFACfg.ini``:
 
+* com1Exe - path to com1DFA executable
 * flagOut - print full model output
 * flagEnt - include entrainment areas
 * flagRes - include resistance areas
+* releaseScenario - name of release area scenario shapfile (with or without extension -shp, if multiple, separate by '_')
```
[the remark this answers is not in the document]
Comment (awirb) [reply docs/moduleCom1DFA.rst:38]: due to the way makeSimDict works, the release area scenarios are not 'allowed' to have an underscore, this is also described in the docu

File: docs/moduleOut3Plot.rst [hunk 2]
```diff
@@ -102,3 +74,73 @@ To run
...
+in1DataPlots can be used to produce plots of a sample and its characteristics derived with ``computeFromDistribution.py``,
+such as: cumulative distribution function (CDF), bar plot of sample values, probability density function (PDF) of the sample,
+comparison plot of empirical- and desired CDF and comparison of empirical- and desired PDF.
+
+
+
+statsPlots
+=================
+
+statsPlots can be used produce scatter plots using a peak dictionary where information on two result parameters of avalanche simulations is saved.
```
[the remark this answers is not in the document]
Comment (awirb) [reply docs/moduleOut3Plot.rst:92]: Done but using is saved

File: docs/moduleOut3Plot.rst [hunk 3]
```diff
@@ -102,3 +74,73 @@ To run
...
+This peak dictionary or a list of those can be created using the function ``extractMaxValues`` of ``ana4Stats/getStats.py``.
+This can be used to visualize results of avalanche simulations where parameter variation has been used or for e.g. in the case of
+multiple release area scenarios. If parameter variation was used to derive the simulation results, the plots indicate the parameter values in color.
+If the input data includes information about the 'scenario' that was used, the plots use different colors for each scenario.
+There is also the option to add a kde plot for each result paramter as marginal plots.
```
[the remark this answers is not in the document]
Comment (awirb) [reply docs/moduleOut3Plot.rst:97]: done

File: docs/moduleCom1DFA.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (awirb) [reply docs/moduleCom1DFA.rst:38]: we could accept any name and  convert it to one not including any '_'

**Review by fso42 — APPROVED** [review]

**Pull request merged**